include/cuco/detail/static_map/functors.cuh

```diff
@@ -84,7 +89,8 @@ struct slot_is_filled {
   template <typename Slot>
   __device__ constexpr bool operator()(Slot const& slot) const noexcept
   {
-    return not cuco::detail::bitwise_compare(empty_sentinel_, thrust::get<0>(slot));
+    return not(cuco::detail::bitwise_compare(empty_sentinel_, thrust::get<0>(slot)) or
+               cuco::detail::bitwise_compare(erased_sentinel_, thrust::get<0>(slot)));
```

Comment:
Unrelated to the PR but FYI: We're using `thrust::get` here which is the right choice. In other places (including my recent PRs), we use `.first`/`.second` which is less generic. We should put this on our TODO list for minor cleanups.